functions/src/CVE/index.ts

```diff
@@ -86,3 +86,23 @@ function getCveWithCveID(id:any,res:any){
 
 //function SPL1and2Helper(id1, id2)
 
+function splStartHelper(id : string, res : any) : void {
+  var db = admin.database();
+  var ref = db.ref('/CVEs');
+
+  ref.on("value", function(snapshot) {
+    let cves = snapshot.val();
+    let cve_array : Array<any> = [];
+    const cve_jsons : any = Enumerable.from(cves)
+      .where(function(obj) {return obj.value['ASB'] < id})
+      .select(function (obj){
+        return obj.value;})
+  for (const cve of cve_jsons){
+    cve_array.push(cve);
+  }
+  const result = {
```

Comment:
I think that this might need a change, let's discuss tomorrow. We should cut off the CVEs for a given Android version (otherwise we return huge, no so useful set). We have Android versions release dates in the database.